src/stories/Moviegrid.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import "./moviegrid.css"
+export function MovieGrid(props) {
+    return (<div className="moviegrid">
+        {props.listMovie.map(e => {
+            return (<div>{e}</div>)
+        })}
```

Comment:
Peut-etre simplifié si désiré : 
```suggestion
        {props.listMovie.map(e => <div>{e}</div>)}
```

Reply:
Plus efficace en plus. Evite le return imbriqué grâce au rendu de l'opérateur map de. type PropNode de sous type htmlelement